io/post_fv3.F90

```diff
@@ -4343,6 +4340,28 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
         end do
       end do
 
+      do l=lm,1,-1
+!$omp parallel do default(none) private(i,j) shared(l,jsta,jend,omga,wh,dpres,zint,alpint,q,t,spval,ista,iend)
+        do j=jsta,jend
+          do i=ista,iend
+            if(wh(i,j,l) /= spval) then
+              if (omga(i,j,l) == spval .and. dpres(i,j,l) /= spval .and. zint(i,j,l) /=spval)  &
+                  omga(i,j,l) = (-1.) * wh(i,j,l) * dpres(i,j,l)/zint(i,j,l)
+              zint(i,j,l) = zint(i,j,l) + zint(i,j,l+1)
```

Comment:
What will happen on this line if `zint(i,j,l) == spval`?

Reply:
added the conditions to ensure zint(i,j,l) and zint(i,j,l+1) is not spval.
I will re-run all regressions tests now if there are no more comments.